demos/checkbox.html

```diff
@@ -201,6 +202,10 @@
         document.getElementById('make-ind').addEventListener('click', function() {
           checkboxInstance.indeterminate = true;
         });
+
+        document.getElementById('toggle-disabled').addEventListener('click', function() {
```

Comment:
Aha! You are correct. Would you actually mind adding that logic to the `setDisabled` method in the foundation? It would look something like:

1. Add `mdc-checkbox--disabled` to `cssClasses` in `constants.js`
1. Then in the `foundation` add this line to the the `setDisabled()` method

```js
  setDisabled(disabled) {
    this.getNativeControl_().disabled = disabled;
    if (disabled) {
      this.adapter_.addClass(cssClasses.DISABLED);
    } else {
      this.adapter_.removeClass(cssClasses.DISABLED);
    }
  }
```
^ Something like that


I believe that should do it, and then your demo code should work 👍